.github/workflows/backend.yml

```diff
@@ -34,7 +36,10 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: setup env
+    - name: Install rust toolchain
+      uses: dtolnay/rust-toolchain@stable
+
```

Comment:
[turns out the default image already comes with the rust toolchain](https://github.com/Swatinem/rust-cache)
but with now what i've realised is that sometimes github would be behind on rust updates. this can be kept in